test/rules/structured-imports/ancestorCases-fix.ts.lint

```diff
@@ -0,0 +1,6 @@
+import * as SomethingAbsolute from 'something-absolute';
```

Comment:
I added complete gibberish to this file, and the test still passed.  I'm guessing that `-fix` tests aren't meaningful until a rule uses`'hasFix': true`